From 4a00f667612acc0df591f05a9ef4537554b66acb Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Fri, 25 Sep 2020 02:01:34 -0700
Subject: [PATCH 1669/1852] ASoC: xlnx: formatter: Fix warnings for
 PW.SET_BUT_NOT_USED

commit 9a382faf6b41a78d44bea1aba0b49241dd3f8ca2 from
https://github.com/Xilinx/linux-xlnx.git

This patch fix the warnings for the PW.SET_BUT_NOT_USED type.

The actual warnings for below functions are:

for xlnx_formatter_pcm_hw_params():

Event set_but_not_used: variable "sample_rate" was set but never used.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 sound/soc/xilinx/xlnx_formatter_pcm.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/sound/soc/xilinx/xlnx_formatter_pcm.c b/sound/soc/xilinx/xlnx_formatter_pcm.c
index 277637c13a80..a2eef3c70a45 100644
--- a/sound/soc/xilinx/xlnx_formatter_pcm.c
+++ b/sound/soc/xilinx/xlnx_formatter_pcm.c
@@ -447,7 +447,7 @@ static int xlnx_formatter_pcm_hw_params(struct snd_pcm_substream *substream,
 					struct snd_pcm_hw_params *params)
 {
 	u32 low, high, active_ch, val, bits_per_sample, bytes_per_ch;
-	u32 aes_reg1_val, aes_reg2_val, sample_rate;
+	u32 aes_reg1_val, aes_reg2_val;
 	int status;
 	u64 size;
 	struct pl_card_data *prv;
@@ -464,7 +464,6 @@ static int xlnx_formatter_pcm_hw_params(struct snd_pcm_substream *substream,
 
 	adata = dev_get_drvdata(component->dev);
 
-	sample_rate = params_rate(params);
 	active_ch = params_channels(params);
 	if (active_ch > stream_data->ch_limit)
 		return -EINVAL;
-- 
2.31.1

